src/serialization/serialization.h

```diff
@@ -158,18 +169,9 @@ inline bool do_serialize(Archive &ar, bool &v)
  * VARINT_FIELD_F(). Otherwise, this macro is similar to
  * BEGIN_SERIALIZE_OBJECT(), as you should list only field serializations.
  */
-#define BEGIN_SERIALIZE_OBJECT_FN(stype)               \
-  template <bool W, template <bool> class Archive>     \
-  bool do_serialize_object(Archive<W> &ar, stype &v);  \
-  template <bool W, template <bool> class Archive>     \
-  bool do_serialize(Archive<W> &ar, stype &v) {        \
-    ar.begin_object();                                 \
-    bool r = do_serialize_object(ar, v);               \
-    ar.end_object();                                   \
-    return r;                                          \
-  }                                                    \
-  template <bool W, template <bool> class Archive>     \
-  bool do_serialize_object(Archive<W> &ar, stype &v) { \
+#define BEGIN_SERIALIZE_OBJECT_FN(stype, ...)                                           \
```

Comment:
After the first argument, you have to list the typename and variable name? A little funky, but it works.

Reply:
Yup!